testutils/server/data.server.testutil.js

```diff
@@ -65,34 +65,19 @@ async function saveReferences(_docs, done = () => {}) {
 }
 
 /**
- * Clear specified database collections
- * @param {string[]} collections - array of collection names to have all documents removed
+ * Clear all collections in a database
+ * Usage in mocha: afterEach(clearDatabase)
  * @returns {Promise<void>}
  */
-async function clearDatabaseCollections(collections) {
+async function clearDatabase() {
```

Comment:
These changes could actually be done in separate PR to not wait for fixing failing tests here.